cairo/tests/ef_tests/test_rlp.py

```diff
@@ -56,33 +53,44 @@ def ethtest_fixtures_as_pytest_fixtures(
     return pytest_fixtures
 
 
+@pytest.fixture(scope="module")
+def cairo_filepath():
+    return Path(f"{Path().cwd()}/cairo/ethereum_rlp/rlp.cairo")
+
+
 @pytest.mark.parametrize(
     "raw_data, expected_encoded_data",
     ethtest_fixtures_as_pytest_fixtures("rlptest.json"),
 )
 def test_ethtest_fixtures_for_rlp_encoding(
-    raw_data: Extended, expected_encoded_data: Bytes
+    raw_data: Extended,
+    expected_encoded_data: Bytes,
+    cairo_filepath,
```

Comment:
not used

Reply:
required because the parametrized function returns two args